SessionPlayer.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { View, StyleSheet } from 'react-native';
+
+import PlayerActionButton from 'PlayerActionButton';
+
+/*
+ * Guides
+ */
+import GuideBreathingNormal from 'Guides/Breathing/Normal';
+import GuideBreathingPause from 'Guides/Breathing/Pause';
+import GuideBreathingFinished from 'Guides/Breathing/Finished';
+
+import GuidePowerPoseSuperHuman from 'Guides/PowerPose/SuperHuman';
+import GuidePowerPosePause from 'Guides/PowerPose/Pause';
+import GuidePowerPoseFinished from 'Guides/PowerPose/Finished';
+
+const guides = {
+    Breathing: {
+        Normal: GuideBreathingNormal,
+        Paused: GuideBreathingPause,
+        Finished: GuideBreathingFinished,
+    },
+    PowerPose: {
+        SuperHuman: GuidePowerPoseSuperHuman,
+        Paused: GuidePowerPosePause,
+        Finished: GuidePowerPoseFinished,
+    }
+}
+
+export default class SessionPlayer extends Component {
+    constructor(props) {
+        super(props);
+
+        this.session =
+            [ // A Session
+                { // Guides
+                    guideName: 'Breathing',
+                    instructions: [ // Instructions
+                        { // Descriptor
+                            type: 'Normal'
```

Comment:
Get rid of `type` property and rename `Normal`, `PowerPose`, etc into "Playing".